src/common/java/net/minecraftforge/gradle/common/util/RunConfig.java

```diff
@@ -77,6 +79,7 @@ public RunConfig(final Project project, final String name) {
         this.name = name;
 
         this.mods = project.container(ModConfig.class, modName -> new ModConfig(project, modName));
+        this.folderName = project.getObjects().property(String.class).convention(project.getName());
```

Comment:
Why this convention?

Reply:
Because I think that the default folder name should be the project's name.